app/src/main/java/de/danoeh/antennapod/activity/VideoplayerActivity.java

```diff
@@ -94,18 +96,18 @@ protected void onResume() {
 
     @Override
     protected void onPause() {
-        super.onPause();
         videoControlsHider.stop();
         if (controller != null && controller.getStatus() == PlayerStatus.PLAYING) {
             controller.pause();
         }
+        super.onPause();
     }
 
     @Override
     protected void onDestroy() {
-        super.onDestroy();
         videoControlsHider.stop();
         videoControlsHider = null;
+        super.onDestroy();
```

Comment:
This isn't written on stone, but it's a good principle that makes sense to me and I try to follow
http://stackoverflow.com/questions/16925579/android-implementation-of-lifecycle-methods-can-call-the-superclass-implementati